core/domain/html_validation_service_test.py

```diff
@@ -1248,3 +1248,10 @@ def test_no_convertion_of_non_interactive_image_tags(self):
                 html_validation_service.convert_svg_diagram_tags_to_image_tags(
                     test_case['html_content']),
                 test_case['expected_output'])
+
+    def test_fix_incorrectly_encoded_chars(self):
```

Comment:
Could you write a test function name for this that's more in line with https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests ?

Reply:
Done!